Makefile

```diff
@@ -214,8 +214,19 @@ dev/syncdeps:
 
 .PHONY: release/versionbump
 release/versionbump:
+	bazel run //hack/versionbump:versionbump -- patch $(VERSION) > $(PWD)/version.txt
+	# Reload make so it rereads version.txt
+	$(MAKE) release/gen-files
+
+.PHONY: release/gen-files
+release/gen-files:
+	for target in manifests/operator.yaml manifests/patches/deployment_patch.yaml \
```

Comment:
Why not do this loop and writing directly in `hack/crdbversions/main.go`?

Reply:
That's done on purpose to keep the logic simple. It can be done by `hack/crdbversions/main.go`, but then you would need to either hardcode the file list or pass some kind of a structure with input and output destinations.

Reply:
Fair, we have a hard coded list here or somewhere else though, no? I'm not too fussy about this just would like to minimize the amount of "logic" in the Makefile but that's probably more personal preference.

Reply:
Not sure if I'm happy with https://github.com/cockroachdb/cockroach-operator/pull/550/commits/7d9d42d299a2404a8a359be8e722965ad3449da5 TBH... If feels less Unix-way :)